src/plugins/discover/public/components/discover_grid/constants.ts

```diff
@@ -11,10 +11,10 @@ import { EuiDataGridStyle } from '@elastic/eui';
 // data types
 export const kibanaJSON = 'kibana-json';
 export const GRID_STYLE = {
-  border: 'all',
+  border: 'horizontal',
```

Comment:
We may want to hold off on removal of vertical borders until we add the [proposed column highlighting](https://www.figma.com/file/zxhZnjEOmgodwQwQlkZ6pZ/Data-grid-improvements-v2?type=design&node-id=181%3A587879&mode=design&t=a0Rm7Gji4IL10CaH-1), as it helps to indicate to users where the column boundaries are located when vertical borders are absent. Thoughts?

Reply:
I don't think we should hold off on this change until column highlighting is available. Padding included in the cells should produce enough space between columns for users to tell when a column begins and ends. See:
<img width="838" alt="image" src="https://github.com/elastic/kibana/assets/4016496/c5fbdf80-70aa-43ed-babc-fce2b8e137d7">

Also, note that the column highlighting is not currently considered part of the MVP we're aiming for.

Reply:
Agreed we don't need to wait, IMO it's already looking nice 👍